test/notebooks/notebookTestUtils.ts

```diff
@@ -0,0 +1,90 @@
+import { ElementHandle, Frame } from "puppeteer";
```

Comment:
Remove this file and inline these all in the test file. I think there is no benefit to abstracting this, especially for end to end tests. It adds another layer of indirection. Sandi Metz articulates my opinions on this very well https://sandimetz.com/blog/2016/1/20/the-wrong-abstraction

Reply:
By inlne, I mean make the test file one big function. No need to have these all broken up into small functions.

Reply:
My thought was, for any notebook UI test, it should be framed as 
upload notebook -> do something (open / run / change kernel and run etc) -> delete notebook 
Hence this abstraction. Upload notebook and delete notebook should be called in the setup and cleanup for a class of tests. I will add one more test to this PR to make that clear.